clang/include/clang/Basic/LangOptions.h

```diff
@@ -444,6 +444,19 @@ class LangOptionsBase {
     IncompleteOnly = 3,
   };
 
+  enum class ArrayBoundsStrictFlexArraysLevelKind {
```

Comment:
To match numerical values, less mental efforts for readers.

```
  enum class ArrayBoundsStrictFlexArraysLevelKind {
    // Use same StrictFlexArrayLevel as compiler.
    None = -1,
    /// Any trailing array member is a FAM.
    Default = 0,
    /// Any trailing array member of undefined, 0, or 1 size is a FAM.
    OneZeroOrIncomplete = 1,
    /// Any trailing array member of undefined or 0 size is a FAM.
    ZeroOrIncomplete = 2,
    /// Any trailing array member of undefined size is a FAM.
    IncompleteOnly = 3,
  };
```

Reply:
we can't use -1 because it's a bitfield, but I can use 4

Reply:
any way to avoid None? and new enum completely?
E.g. Driver copy from another flag if this one is missing?

Reply:
because we currently only store 3 bits for this, using -1 would be out of range of that and just truncate to 111 = 7

Reply:
i have a weak preference for flags to have a value that can represent the state that is the same as if they were not passed in. that is of course also possible in the driver, but would be a bit more complexity there.